fvm/transaction.go

```diff
@@ -40,6 +43,19 @@ func (proc *TransactionProcedure) SetTraceSpan(traceSpan opentracing.Span) {
 
 func (proc *TransactionProcedure) Run(vm *VirtualMachine, ctx Context, st *state.StateHolder, programs *programs.Programs) error {
 
+	defer func() {
+		if r := recover(); r != nil {
+
+			if strings.Contains(fmt.Sprintf("%v", r), "[Error Code: 1106]") {
+				ctx.Logger.Error().Str("trace", string(debug.Stack())).Msg("VM LedgerIntractionLimitExceeded panic")
+				proc.Err = errors.NewLedgerIntractionLimitExceededError(state.DefaultMaxInteractionSize, state.DefaultMaxInteractionSize)
+				return
+			}
```

Comment:
I think we should take the time to fix this properly, the string comparison seems a bit risky to me